gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -732,50 +721,12 @@ function VOICE.CanSpeak()
     return client.voice_battery > VOICE.battery_min or IsRoleChatting(client)
 end
 
-local speaker = surface.GetTextureID("voice/icntlk_sv")
-
----
--- Draws a popup displaying the speaking @{Player}s
--- @param Player client This should be the local @{Player}
--- @return boolean
--- @realm client
--- @internal
-function VOICE.Draw(client)
-    local b = client.voice_battery
-
-    if not b or not VOICE.battery_max or b >= VOICE.battery_max then
-        return
-    end
-
-    local x, y = 25, 10
-    local w, h = 200, 6
-
-    if b < VOICE.battery_min and CurTime() % 0.2 < 0.1 then
-        surface.SetDrawColor(200, 0, 0, 155)
-    else
-        surface.SetDrawColor(0, 200, 0, 255)
-    end
-
-    surface.DrawOutlinedRect(x, y, w, h)
-
-    surface.SetTexture(speaker)
-    surface.DrawTexturedRect(5, 5, 16, 16)
-
-    x = x + 1
-    y = y + 1
-    w = w - 2
-    h = h - 2
-
-    surface.SetDrawColor(0, 200, 0, 150)
-    surface.DrawRect(x, y, w * math.Clamp((client.voice_battery - 10) / 90, 0, 1), h)
-end
-
 ---
 -- This hook can be used to modify the background color of the voice chat
--- box that is rendered on the client.
+-- box that is rendered on the client. This is done by setting the voice chat mode.
 -- @param ply The player that started a voice chat
--- @param Color clr The color that is used if this hook does not modify it
--- @return Color The new and modified color
+-- @param number mode The voice chat mode that is used if this hook does not modify it
+-- @return number The new and modified mode
 -- @hook
 -- @realm client
-function GM:TTT2ModifyVoiceChatColor(ply, clr) end
```

Comment:
This needs to be included in the changelog

Reply:
done, listed as breaking change